lightwood/api/data_source.py

```diff
@@ -393,6 +391,40 @@ def _prepare_encoders(self):
                         except KeyError:
                             config['depends_on_column'] = [d['name']]
 
+        for config in self.config['output_features']:
+            column_name = config['name']
+            column_data = self.get_column_original_data(column_name)
+
+            prev_col = [c for c in self.config['input_features'] if c['name'] == f'__mdb_ts_previous_{column_name}']
+            if prev_col:
+                # 0-indexed as we know it can only be one
+                config['depends_on_column'] = [prev_col[0]['name']]
+                config['encoder_attrs']['normalizers'] = prev_col[0]['additional_info']['normalizers']
+                config['encoder_attrs']['group_combinations'] = prev_col[0]['additional_info']['group_combinations']
+                config['encoder_attrs']['normalize_by_group'] = True
```

Comment:
Why not default this to `False`, as it stands right now it's only turned on for autorgressive timeseries it seems... which doesn't really make sense to me, seems it would work just as well for all timeseries.

Reply:
Turns out this flag is a leftover from when there was no `TsNumericEncoder`, removing